templates/types/extractor/fastapi/extractor/pages/index.py

```diff
@@ -0,0 +1,40 @@
+import reflex as rx
+
+from ..components import (
+    SchemaState,
+    UploadedFilesState,
+    extract_data_component,
+    schema_editor_component,
+    upload_component,
+)
+from ..templates import template
+
+
+@template(
+    route="/",
+    title="Upload",
+    on_load=[
+        SchemaState.init_schema,
+        UploadedFilesState.load_files,
+    ]
+)
+def upload() -> rx.Component:
+    """The upload page.
```

Comment:
```suggestion
def index() -> rx.Component:
    """The main index page.
```

why is it called upload page if it's the only page?